parameters/world_modeling/world_model_gmapping.yaml

```diff
@@ -17,8 +17,8 @@ plugins:
   parameters:
       robot_name: amigo
 
-- name: navigation
-  lib: libed_navigation_plugin.so
+- name: kinect_navigation
```

Comment:
Why don't we need the navigation plugin anymore? What happens if we swipe objects into the world model?